### PR TITLE
H-149: Hide hotkey cheat sheet

### PR DESCRIPTION
**Pull request by CiaranMn**
## 🌟 What is the purpose of this PR?

There's a command bar which appears if you hit cmd/ctrl + K. There's also a hotkey cheat sheet which appears if you press ?. This happens even if you press ? inside contenteditables, which is not the desired behaviour.

This PR fixes that bug, but also hides the cheat sheet entirely, as it doesn't serve much purpose at the moment, since the hot keys are already shown on the command bar. Its use would be to be able to display _all_ hot keys, even ones which are not applicable in the current context, but since we don't have many at the moment and the cheat sheet requires styling, it's not worth it.

### 🚢 Has this modified a publishable library?

This PR:

- [x] does not modify any publishable blocks or libraries, or modifications do not need publishing

### 📜 Does this require a change to the docs?

The changes in this PR:

- [x] are internal and do not require a docs change

### 🕸️ Does this require a change to the Turbo Graph?

The changes in this PR:

- [x] do not affect the execution graph

### CODE REVIEW

File: apps/hash-frontend/src/shared/layout/layout-with-sidebar/sidebar.tsx [hunk 1]
```diff
@@ -68,7 +68,6 @@ export const PageSidebar: FunctionComponent = () => {
         icon={faHome}
         title="Home"
         href="/"
-        tooltipTitle="View your inbox and latest activity"
```
Comment (CiaranMn) [inline apps/hash-frontend/src/shared/layout/layout-with-sidebar/sidebar.tsx:71]: Driveby: remove inaccurate tooltip

**Review by nonparibus — DISMISSED** [review]

**Review by nonparibus — APPROVED** [review]
